src/CoreBundle/Event/RecordChangeEvent.php

```diff
@@ -15,6 +15,8 @@ class RecordChangeEvent extends Event
      */
     private $recordId;
 
+    private string $cmsTblConfId;
```

Comment:
rename $tableId property and in constructor to $tableName, add mehod getTableName() and deprecate GetTableId() method. Call getTableName in GetTableId.

Reply:
Done.